containers/datalab/content/run.sh

```diff
@@ -56,6 +56,13 @@ if [[ -n "${GATEWAY_VM}" ]]; then
   fi
 fi
 
+if [[ "${CLI_LOGIN}" == "true" ]]; then
+  USER_EMAIL=`gcloud auth list --format="value(account)"`
+  if [[ -z "${USER_EMAIL}" ]]; then
+    gcloud auth login
+  fi
+fi
```

Comment:
@qimingj I've just gone ahead and split this out to a helper method, so that we have both flows operate the exact same way.

That should address both of your comments.